src/bench/bench.cpp

```diff
@@ -8,97 +8,120 @@
 #include <assert.h>
 #include <iostream>
 #include <iomanip>
-#include <sys/time.h>
+#include <chrono>
+#include <algorithm>
+#include <regex>
 
-benchmark::BenchRunner::BenchmarkMap &benchmark::BenchRunner::benchmarks() {
-    static std::map<std::string, benchmark::BenchFunction> benchmarks_map;
-    return benchmarks_map;
+void benchmark::ConsolePrinter::header()
+{
+    std::cout << "# Benchmark, evals, iterations, total, min, max, median" << std::endl;
```

Comment:
This is wrong for is_list_only - maybe is_list_only mode should also call result() with some other parameter for the mode?

Reply:
fixed in the following commit